app/server/utils/ussd/token_processor.py

```diff
@@ -56,44 +75,73 @@ def transfer_token(sender: User, recipient: User, amount: float, reason_id: Opti
 
     @staticmethod
     def send_balance_sms(user: User):
-        balance = TokenProcessor.get_balance(user)
-        # TODO: send sms
+        def get_token_info(transfer_account: TransferAccount):
+            token = transfer_account.token
+            limit = TokenProcessor.get_limit(user, token)
+            return {
+                # TODO: is it name or symbol?
+                "name": token.name,
+                "balance": transfer_account.balance,
+                "limit": limit.transfer_balance_percentage,
+                "exchange_rate": TokenProcessor.get_exchange_rate(user, token)
+            }
+        # TODO: should we merge transfer accounts with same tokens..? any currencies we should filter out?
+        token_info = map(get_token_info, user.transfer_accounts)
+        token_balances = "\n".join(map(lambda x: f"{x['name']} {x['balance']}", token_info))
+        # this one is also not well generalized currently - how get real world currency name of reserve token?
```

Comment:
on the system, we won't call the reserve token `DAI` or whatever - we'll just call it eg USD, so can be called by the symbol just as the other ones are